mcp_server/quantum_tools.py

```
@@ -14,444 +14,469 @@
         except Exception as e:
             logger.error(f"Resource management failed: {e}")
-            return {
-                'success': False,
-                'error': str(e)
-            }
-    
-    def _llm_to_optimization_problem(self, training_data: Dict[str, Any], model_config: Dict[str, Any]) -> Dict[str, Any]:
+            return {"success": False, "error": str(e)}
+
+    def _llm_to_optimization_problem(
+        self, training_data: Dict[str, Any], model_config: Dict[str, Any]
```

Ruff (0.11.9)

Unused method argument: `training_data`

(ARG002)